### PR TITLE
Stackdriver logging and monitoring

### LINKED ISSUE #40: Enable new Stackdriver logging and monitoring
Add specification of logging and monitoring with Stackdriver

```
gke_logging_service = "logging.googleapis.com/kubernetes"
```

https://www.terraform.io/docs/providers/google/r/container_cluster.html#logging_service

### PR DESCRIPTION
**Pull request by wwwil**
Add logging and monitoring services specification to use new Kubernetes Stackdriver integration.

Closes #40.

### CODE REVIEW

**Comment by wwwil** [comment]
Putting this back to WIP, gonna hold off until we've jumped to Terraform v0.12 syntax.

File: variables.tf [hunk 1]
```diff
@@ -186,3 +186,23 @@ Defines up to 20 external networks that can access Kubernetes master
...
+EOF
+}
+
+variable "stackdriver_monitoring" {
+  type    = "string"
+  default = "true"
+
+  description = <<EOF
+Whether Stackdriver Kubernetes montioring is enabled. This should only be set to
+"false" if another montioring solution is set up.
```
Comment (charlieegan3) [inline variables.tf]:
```suggestion
"false" if another monitoring solution is set up.
```

File: variables.tf [hunk 2]
```diff
@@ -186,3 +186,23 @@ Defines up to 20 external networks that can access Kubernetes master
...
+"false" if another logging solution is set up.
+EOF
+}
+
+variable "stackdriver_monitoring" {
+  type    = "string"
+  default = "true"
+
+  description = <<EOF
+Whether Stackdriver Kubernetes montioring is enabled. This should only be set to
```
Comment (charlieegan3) [inline variables.tf]:
```suggestion
Whether Stackdriver Kubernetes monitoring is enabled. This should only be set to
```

**Comment by wwwil** [comment]
@charlieegan3 Thanks for the review!

**Pull request merged**